fastly/data_source_fastly_tls_activation_ids_test.go

```diff
@@ -0,0 +1,107 @@
+package fastly
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/stretchr/testify/require"
+	"strings"
+	"testing"
+)
+
+func TestAccDataSourceFastlyTLSActivationIds_basic(t *testing.T) {
+	domain := fmt.Sprintf("tf-test-%s.com", acctest.RandomWithPrefix("tf-test-"))
```

Comment:
If I'm understanding the `acctest.RandomWithPrefix` function, I presume what we actually want is...

```suggestion
	domain := fmt.Sprintf("%s.com", acctest.RandomWithPrefix("tf-test-"))
```

...as the random name we're generating will already have `"tf-test-"` as part of the prefix.

Reply:
Same comment applies in other test files.